### ISSUE TITLE
Backend quit unexpectedly

### ISSUE THREAD

**Issue by yungyazzy** [divamgupta/diffusionbee-stable-diffusion-ui#464]
Hey everyone!
Downloaded DiffusionBee and tried to start using. After trying to generate I have this problem. Does anyone knows how to fix this?
<img width="287" alt="error" src="https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/assets/142102353/2daf59e0-2f20-43ce-ad9b-97ce1d1685fa">

**Comment by agriffith** [comment]
I am also seeing this behavior.

**Comment by jabevan** [comment]
Also getting this error on DiffusionBee 2.4.3:

Backend quit unexpectedly. rosetta error: unable to mmap __TEXT: 1 
/var/db/oah/f64a632c03f1e4b34d05b34c4b2d5a013fed14d8a3fd186205f866dc875f6d71/373be89c46959e1a7f89eb8a077213cb90cdb40807b7a7757edea018152bc644/libffi.8.dylib.aot

Running Sonoma 14.2.1

**Comment by divamgupta** [comment]
You might be trying to run intel version on m1. Could you check and
confirm?

**Comment by jabevan** [comment]
You’re right. Very amateur mistake. I was trying to run the Intel version on M1.
